File: Samples-Java/Samples/src/main/java/fragments/dataform/DataFormValidationFragment.java
package fragments.dataform;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.telerik.android.sdk.R;
import com.telerik.widget.dataform.engine.NonEmptyValidator;
import com.telerik.widget.dataform.visualization.RadDataForm;

import activities.ExampleFragment;

public class DataFormValidationFragment extends Fragment implements ExampleFragment {
    @Override
    public String title() {
        return "Validation";
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        ViewGroup layoutRoot = (ViewGroup) inflater.inflate(R.layout.fragment_dataform_validation, container, false);

        RadDataForm dataForm = (RadDataForm)layoutRoot.findViewById(R.id.data_form_validation);
        dataForm.setEntity(new Person());

        dataForm.getExistingEditorForProperty("Name").property().setValidator(new NonEmptyValidator());

        return layoutRoot;
    }
}

File: Samples-Xamarin/Fragments/Dataform/DataFormValidationFragment.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Util;
using Android.Views;
using Android.Widget;

using Com.Telerik.Widget.Dataform.Visualization;
using Com.Telerik.Widget.Dataform.Engine;
using Com.Telerik.Widget.Dataform.Visualization.Editors;

namespace Samples
{
	public class DataFormValidationFragment : Android.Support.V4.App.Fragment, ExampleFragment
	{
		public override View OnCreateView (LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
		{
			ViewGroup layoutRoot = (ViewGroup) inflater.Inflate(Resource.Layout.fragment_dataform_validation, container, false);

			RadDataForm dataForm = (RadDataForm)layoutRoot.FindViewById(Resource.Id.data_form_validation);
			dataForm.SetEntity (new Person ());

			dataForm.GetExistingEditorForProperty ("Name").Property ().Validator = new NonEmptyValidator();

			return layoutRoot;
		}

		public string Title() {
			return "Validation";
		}
	}
}
